### ISSUE TITLE
Error when importing

### ISSUE THREAD

**Issue by andrewkouri** [docusealco/docuseal-react#1]
```
import { DocusealForm } from '@docuseal/react'
```

```
Could not find a declaration file for module '@docuseal/react'. '/node_modules/@docuseal/react/dist/index.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/docuseal__react` if it exists or add a new declaration (.d.ts) file containing `declare module '@docuseal/react';`ts(7016)
```
Using typescript on NextJS 13

**Comment by AlexBTurchyn** [comment]
@starlight-akouri thanks for pointing this out, that's true that there is an issue with TS types definition - will try to get this resolved asap.

**Comment by AlexBTurchyn** [comment]
@starlight-akouri it has been fixed in the latest 1.0.8 release. Can you please confirm that the updated npm package works with your TS project?
